Anastasiya Santalova/grid/SCSS/hw2_grid.scss

```diff
@@ -0,0 +1,317 @@
+@import "_variables";
```

Comment:
Even though the filename starts with underscore, you should still import external files without it, otherwise SASS will fail during compilation